### ISSUE TITLE
Update Bars style, pinned files and more

### ISSUE THREAD

**Issue by cbasdev** [cbasdev/dracula-purple-theme#7]
### Inspiration:
<img width="145" alt="image" src="https://user-images.githubusercontent.com/33915497/214860215-7b823f6f-4443-476e-9e75-bff655f2c7be.png">

```typescript
{
  "workbench.colorCustomizations": {
    "activityBar.background": "#00AA00"
  }
}
```

When change colors and styles, the PR has been aproved by me. The colors should have a purple hue.

**Comment by andrea12mart** [comment]
Does this issue require interactivity? directly access the jscode 😢

**Comment by cbasdev** [comment]
Yes, this is one of the problems why it was not updated until vscode had support for interactive styles
